HW1/P3/P3.py

```diff
@@ -0,0 +1,19 @@
+# Ankit Gupta
+# CS 205
+import pyspark
+sc = pyspark.SparkContext(appName="Spark1")
+import numpy as np 
+import itertools
+
+
+# Returns all orderings of a string
+# These are not necessarily all valid words
+def get_anagrams(s):
+	return ["".join(perm) for perm in itertools.permutations(s)]
+
+wlist = sc.textFile('../P3/EOWL_words.txt')
+
+sorted_letter_to_word = wlist.map(lambda x: (''.join(sorted(x)), x))\
+                            .groupByKey().map(lambda (x, y): (x, len(y), list(y)))\
+                            .sortBy(lambda x: (-1)*x[1])
```

Comment:
It would be helpful to add some comments about lines 17 and 18 so that it's clear how the data is being transformed and why, but this code is very clean so it's not that necessary